extensions/amp-story-subscriptions/0.1/test/test-amp-story-subscriptions.js

```diff
@@ -1,37 +1,75 @@
 import '../amp-story-subscriptions';
-import {htmlFor} from '#core/dom/static-template';
+import * as Preact from '#core/dom/jsx';
+
+import {afterRenderPromise} from '#testing/helpers';
+
+import {registerServiceBuilder} from '../../../../src/service-helpers';
+import {
+  Action,
+  AmpStoryStoreService,
+} from '../../../amp-story/1.0/amp-story-store-service';
+import {AmpStorySubscriptions} from '../amp-story-subscriptions';
 
 describes.realWin(
   'amp-story-subscriptions-v0.1',
   {
     amp: {
       runtimeOn: true,
-      extensions: ['amp-story-subscriptions:0.1'],
+      extensions: ['amp-story:1.0', 'amp-story-subscriptions:0.1'],
     },
   },
   (env) => {
     let win;
     let doc;
-    let html;
+    let subscriptionsEl;
+    let storeService;
+    let storySubscriptions;
+
+    const nextTick = () => new Promise((resolve) => win.setTimeout(resolve, 0));
 
     beforeEach(() => {
       win = env.win;
       doc = win.document;
-      html = htmlFor(doc);
+
+      storeService = new AmpStoryStoreService(win);
+      registerServiceBuilder(win, 'story-store', function () {
```

Comment:
A common pattern is something like this:
```js
import {Services} from ....
storeService = new AmpStoryStoreService(win);
env.sandbox.stub(Services, 'storyStoreServiceForOrNull').returns(storeService)
```

Reply:
Done.